src/js/media-controller.js

```diff
@@ -132,6 +135,10 @@ class MediaController extends MediaContainer {
       //   - Element.requestFullscreen()
       //
       MEDIA_ENTER_FULLSCREEN_REQUEST: () => {
+        if (!document.fullscreenEnabled) {
+          console.warn('Fullscreen support is disabled, not entering fullscreen');
```

Comment:
can use help wordsmithing this.

Reply:
suggestion(non-blocking): `"Fullscreen support is unavailable; not entering fullscreen"`? also is perfectly fine as is imo.